simpa/utils/libraries/molecule_library.py

```diff
@@ -417,6 +457,24 @@ def append(self, value: Molecule = None, key: str = None):
         self.molecular_composition_dictionary[key] = value
         return self
 
+    def append_filler(self, value: Molecule = None, key: str = None):
+        """
+        Behaves the same as append in principle. However, the volume fraction of the filler molecule will be ignored.
+        Instead it will be assigned a value between 0 and 1 to make the total volume fraction equal to 1.
+
+        You may only define one filler molecule.
+        """
+        if self.filler is not None:
+            raise AssertionError("Only one filler molecule can be defined.")
```

Comment:
The filler is not defined per default.
When the first "filler" molecule is set, then its all fine.
If a second molecule is added as a filler, then it does not work anymore.
I think it is added in the documentation of the function.

Any suggestions how this could be made more explicit?